packages/documentation-site/patternfly-docs/content/accessibility/page/page.md

```diff
@@ -3,30 +3,81 @@ id: Page
 section: components
 ---
 
-The **page** component is used to define the basic layout of a page with either vertical or horizontal navigation. Page layouts are defined using page sections, such as the header, body, and footer of a page. The basic layout of a page depends on whether your page uses vertical or horizontal navigation.
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
 
-A page can be combined with many other components, so to create an accessible page, refer to the accessibility recommendations for each component.
+## Accessibility
 
-**Keyboard users** should be able to navigate through the masthead of the page and into other interactive elements of the page using **Tab** and **Shift + Tab** to move forward and backward through interactive elements. 
+To implement an accessible PatternFly **page**:
 
-**Screen reader users** should be able to navigate through and interact with the masthead of the page using the same interactions as keyboard users. Elements of a page that do not have visible descriptive text, such as toggle icons, should have alternative text for screen readers.
+- Provide a [skip to content](/components/skip-to-content) if more than one page shares the same heading or sidebar content.
+- Give each `nav` element on the page a unique `aria-label`.
+- Ensure there is only 1 `main` element on the page.
+- Ensure each page section has an `aria-labelledby` attribute linked to a heading element within the section.
```

Comment:
This is a really good point, and potentially unrelated to this PR, but are we actually following this recommendation in our Page demos?

Reply:
Can't recall off the top of my head but erring on the side of caution I want to say we may not be. That would require a follow up in Core and React, which I can evaluate which, if any, examples/demos need to be updated and create any necessary followups. Probably need to go through Org as well.

You're referring to the last bullet regarding aria-labelledby, correct?

Reply:
I thought it was an MDN docs page but maybe not, it may have been traversing through VO's rotor menu again. Or one of our a11y pages that mention sections shouldn't be used if there's no label/our example of providing a label for scrollable content.

If we do want to keep this it would probably make sense to update it to say use either aria-labelledby or aria-label if no visible text is present, but it may actually make sense to remove (even if for now)

Reply:
Yeah, it makes sense to do, and I'm always up for adding more/better context to help users. I just hesitate to make recommendations if we aren't already walking that walk, so to speak. 🙂

Reply:
Definitely true. I think my thoughts were get a11y docs up to date and update core/react examples as follow ups according to the updated docs, though it may make more sense to not include stuff that we aren't following yet ourselves.

What do you think about me removing the verbiage about labeling page sections from this PR, opening issues in core/react to add consistent labeling, then once those are closed out we open a followup in Org to add in the verbaige about labeling sections?

Reply:
I think that would make sense to me!